backend/src/prisma/schema.prisma

```diff
@@ -0,0 +1,59 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Location {
+  id           Int       @id @default(autoincrement())
+  name         String
+  type         Int
+  street       String
+  locality     String?
+  place        String
+  district     String?
+  region       String
+  postcode     String?
+  country      String
+  latitude     Float
+  longitude    Float
+  description  String?
+  avgCo2       Float?
+  created_id   Int
+  created_at   DateTime  @default(now())
+}
+
+model Room {
+  id           Int       @id @default(autoincrement())
```

Comment:
No idea, but was wondering if we want to spell out ModelId for each model's primary key. In this case Room would have roomId. Open to whatever though. I don't know what the convention or context is here.

Reply:
I mostly try to avoid typing unnecessarily. In the code, I think it will always be in context already, eg. location.id. And I guess when comparing or connecting values it might look something like `if (location.id = room.locationId)`

In SQL sometimes having the full name can be nice, in that (I think) you can choose to omit the table name if the field names are unambiguous.
eg. `from room 
                inner join location on ... 
     where roomId = locationID` 
vs
     ` from room ... location loc
     where room.locationid = loc.id`

Having written that out I thought I would lean (only slightly tbh) toward leaving as-is, but now I think I'm leaning slightly in favour of your suggestion.

Reply:
What I'm much more interested in discussing is Int IDs vs. something GUID-ish.
I notice that the current Location API exposes IDs as some kind of GUID or hash or something, and I think that might be kind of idiomatic for GraphQL?

Also - I'm noticing that Prisma creates Postgres Ints, and I'm thinking if we stick with numeric keys perhaps BigInt would be preferred. Perhaps I'm overthinking: Int would allow for ~400,000 times as many readings as currently exist. Thoughts?

Reply:
There's a limit on 32 bit ints, which is I'm guessing the default here as it is in most places. I don't think there's such a limit on G/UUIDs. I guess I'm leaning towards the unique ids.

The room.id vs room.roomID issue could also manifest when looking at JSON representations of records, especially if they're not beautified. I don't feel strongly about it, but I feel like I should at least point it out when I see it.

Reply:
Good point about the JSON. Decision made: I'll rename the fields and columns. Thanks for the great feedback!